snappy/dpu-decompress/dpu_decompress.c

```diff
@@ -15,22 +15,43 @@
 /*******************
  * Memory helpers  *
  *******************/
-static uint8_t READ_BYTE(struct in_buffer_context *_i)
+
+/**
+ * Read the next input byte from the sequential reader.
+ *
+ * @param _i: holds input buffer information
+ * @return Byte that was read
+ */
+static inline uint8_t READ_BYTE(struct in_buffer_context *_i)
```

Comment:
I doubt 'inline' has any effect, but it won't hurt. If we had a proper disassembler, this would be easy to check, but we are stuck for now.